src/applications/medical-copays/components/AlertView.jsx

```diff
@@ -2,8 +2,39 @@ import React from 'react';
 import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
 import PropTypes from 'prop-types';
 import Alert from './Alerts';
+import alertMessage from '../../combined-debt-portal/combined/utils/alert-messages';
+import OtherVADebts from './OtherVADebts';
+import {
+  ALERT_TYPES,
+  APP_TYPES,
+  API_RESPONSES,
+} from '../../combined-debt-portal/combined/utils/helpers';
 
-const AlertView = ({ pathname, alertType, error }) => {
+const renderAlert = (alertType, hasDebts) => {
```

Comment:
NITPICK: I see this looks very similar to the renderAleryin the other file. Might be a point to refactor later